pkg/servicetest/testdata/services/docker-compose.solr.yaml

```diff
@@ -1,36 +1,53 @@
-# ddev apache solr recipe file
+# DDev Apache Solr recipe file.
 #
-# To use this in your own project: Copy this file to your project's .ddev folder,
-# and create the folder path .ddev/solr/conf. Then, copy the solr configuration
-# files for your project to .ddev/solr/conf. E.g., using Drupal Search API Solr, 
-# you would copy the solr-conf/5.x/ contents into .ddev/solr/conf. The configuration
-# files must be present before running `ddev start`.
+# To use this in your own project:
+# 1. Copy this file to your project's ".ddev" directory.
+# 2. Create the folder path ".ddev/solr/conf".
+# 3. Copy the Solr configuration files for the appropriate plugin/module to
+#    ".ddev/solr/conf". For example, using Drupal 7's Search API Solr module,
+#    copy the files from its "solr-conf/6.x" directory into ".ddev/solr/conf"
+#    so that a file exists with the path ".ddev/solr/conf/solrconfig.xml".
+#
+# Note: The configuration files must be present before running `ddev start`.
 
 version: '3.6'
 
 services:
-  solr: # This is the service name used when running ddev commands accepting the --service flag
-    container_name: ddev-${DDEV_SITENAME}-solr # This is the name of the container. It is recommended to follow the same name convention used in the main docker-compose.yml file.
-    image: solr:5.4
+  # This is the service name used when running ddev commands accepting the
+  # --service flag.
+  solr:
+    # This is the name of the container. It is recommended to follow the same
+    # name convention used in the main docker-compose.yml file.
+    container_name: ddev-${DDEV_SITENAME}-solr
+    # Controls the version of Solr which is installed.
+    image: solr:6.6
     restart: always
```

Comment:
If we're triaging, let's change this to `no`.

```
    restart: "no"
```

Defaulting to `restart: always` was old DDEV

Reply:
`Defaulting to restart: always was old DDEV`
Sounds like there needs to be a new issue for just that change, so make it on all of the example config files.